apps/mail/components/pricing/comparision.tsx

```
@@ -37,351 +209,119 @@ export default function Comparision() {
-                    Professionals, heavy email users
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
+                      });
+                    }
+                  }}
+                  className="px-6 py-2 bg-white/10 hover:bg-white/20 border border-white/20 rounded-lg text-white font-medium transition-colors"
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)